.drone.yml

```diff
@@ -1765,3 +1790,175 @@ matrix:
       INTEGRATION_TEST_SCRIPT: IntegrationTestPaging.php
       NEED_CORE: true
       NEED_INSTALL_APP: true
+
+    # Release Tarball
+    # acceptance tests - ldap specific
+    # with php 7.2 and core master
```

Comment:
```suggestion
    # with php 7.2 and core 10.2.1
```

Reply:
Is anything happening about this comment change?

Reply:
fixed